tests/plugins/test_bump_release.py

```diff
@@ -111,8 +111,26 @@ def test_missing_labels(self, tmpdir, caplog, labels):
         False
     ])
     @pytest.mark.parametrize('next_release', [
-        {'actual': '1', 'expected': '1'},
-        {'actual': '1', 'expected': '2'},
+        {'actual': '1', 'builds': [], 'expected': '1'},
+        {'actual': '1', 'builds': ['1'], 'expected': '2'},
+        {'actual': '1', 'builds': ['1', '2'], 'expected': '3'},
+        {'actual': '20', 'builds': ['19.1'], 'expected': '20'},
+        {'actual': '20', 'builds': ['20', '20.1'], 'expected': '21'},
+        {'actual': '20.1', 'builds': ['20'], 'expected': '21'},
+        {'actual': '20.1', 'builds': ['20', '20.1'], 'expected': '21'},
+        {'actual': '20.2', 'builds': ['20', '20.1'], 'expected': '21'},
+        {'actual': '20.2', 'builds': ['20', '20.1', '20.2'], 'expected': '21'},
+        {'actual': '20.fc25', 'builds': ['20.fc24'], 'expected': '20.fc25'},
+        {'actual': '20.fc25', 'builds': ['20.fc25'], 'expected': '21.fc25'},
+        {'actual': '20.foo.fc25',
+         'builds': ['20.foo.fc25'],
+         'expected': '21.foo.fc25'},
+        {'actual': '20.1.fc25',
+         'builds': ['20.fc25', '20.1.fc25'],
+         'expected': '21.fc25'},
+        {'actual': '20.1.fc25',
+         'builds': ['20.fc25', '20.1.fc25', '21.fc25'],
+         'expected': '22.fc25'},
```

Comment:
Why wouldn't 20.1.fc25 be possible without 20.fc25? In any case, my point is that it shouldn't transform 20.1.fc25 into 20.fc25.

The test case you added is identical to the previous test case.